sdk/messaging/azservicebus/internal/amqpwrap/amqpwrap.go

```diff
@@ -78,22 +78,22 @@ func (w *AMQPClientWrapper) NewSession(opts ...amqp.SessionOption) (AMQPSession,
 	}
 
 	return &AMQPSessionWrapper{
-		inner: sess,
+		Inner: sess,
 	}, nil
 }
 
 type AMQPSessionWrapper struct {
-	inner *amqp.Session
+	Inner *amqp.Session
```

Comment:
Why is this exported now?

Reply:
I'm wrapping a session in the `azservicebus` package so I need to set it when I create the AMQPSessionWrapper.

I could create a constructor func but it seemed like it'd be more work for me in the long term since we have just the single field.